File: Iniciante/URI 1959.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    
    val N: Long = leitor.nextLong();
    val L: Long = leitor.nextLong();
    
    val P: Long = N * L;
    println(P);
}

File: Iniciante/URI 1959.scala
import java.util.Scanner

object Main {

    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val N = leitor.nextLong()
        val L = leitor.nextLong()
        val P = N * L
        println(P)
    }
	
}